src/test/java/uk/gov/hmcts/reform/lrdapi/service/impl/CourtVenueServiceImplTest.java

```diff
@@ -105,7 +105,7 @@ public void test_RetrieveCourtVenuesByEpimsIDs_OneIdPassed() {
 
         List<LrdCourtVenueResponse> courtVenueResponses =
             courtVenueService
-                .retrieveCourtVenueDetails("123", null,  null, null);
+                .retrieveCourtVenueDetails("123", null,  null, null, null);
```

Comment:
Hi @viswajit-kuna , Should we try replace all these null with Java8 Optional ?

Reply:
Hi @souravroy2021 I have tried this for a couple of hours but could not get this to work. I think I need a bit more time but unfortunately, we don't have it as today is the last day in the sprint. Because the functionality is working fine, can we please ignore this review comment for now? I will try to get this to work if and when I do, I will create a tech debt task and deliver this piece of work.